### ISSUE TITLE
Won't build from Carthage

### ISSUE THREAD

**Issue by drekka** [inloop/Styles#77]
The schemes are shared as expected, but it still won't build. I'm guessing Carthage isn't able to find the project as it's not in the root of the checked out project.

**Comment by jakubpetrik** [comment]
Hi @drekka. Can you please share the error and your configuration?

**Comment by drekka** [comment]
Hi. My Cartfile looks like this:

```
github "Swinject/Swinject"
github "SwInject/SwinjectStoryboard"
github "mxcl/PromiseKit"
github "vhesener/Closures"
github "inloop/Styles"
```

And produces this output:

```
Bit-Shifter:Crux derekclarkson$ carthage update --cache-builds --platform iOS --new-resolver
*** Fetching SwinjectStoryboard
*** Fetching Closures
*** Fetching Swinject
*** Fetching Styles
*** Fetching PromiseKit
*** Checking out PromiseKit at "6.3.4"
*** Checking out Closures at "v0.4"
*** Checking out Swinject at "2.4.1"
*** Checking out Styles at "0.15.9"
*** Checking out SwinjectStoryboard at "2.0.2"
*** xcodebuild output can be found in /var/folders/qn/mdmvqmns1v38s60q7q3my2nc0000gn/T/carthage-xcodebuild.4DR903.log
*** Valid cache found for Closures, skipping build
*** Valid cache found for PromiseKit, skipping build
*** No cache found for Styles, building with all downstream dependencies
*** Valid cache found for Swinject, skipping build
*** Valid cache found for SwinjectStoryboard, skipping build
*** Skipped building Styles due to the error:
Dependency "Styles" has no shared framework schemes for any of the platforms: iOS

If you believe this to be an error, please file an issue with the maintainers at https://github.com/inloop/Styles/issues/new
```

I'm not 100% sure, but I think Carthage only looks in the root directory for project and workspaces to scan for build targets. I've seen the layout you're using in a couple of other places and get the impression it's some CocoaPods template. Usually, projects put their project and workspace files in the root.

**Comment by drekka** [comment]
Ok I think I've figured it out. Carthage was checking out v0.15.9. When I changed the Cartfile to specify the latest v0.24.15-beta it build. So just using "inloop/Styles" without a version checks out an out of date, non-cartage friendly version. Instead it needs

```
github "inloop/Styles" "v0.24.15-beta"
```

P.S. The bundle identifier still appears to be "org.cocoapods.Styles". Probably should be "com.inloop.styles" :-)
